### ISSUE TITLE
ProcessTokenException - how can I debug/invoke a modified figma2flutter

### ISSUE THREAD

**Issue by gronka** [mark-nicepants/figma2flutter#19]
Before anything else - I'd like to debug this myself, but I'm not sure how to execute/invoke a dart "binary" such as figma2flutter. 

If I could get some simple steps, I'd be happy to look into this myself. Something like
1. git clone
2. ???
3. ./myfigma2flutter --input ...

Thanks

### :information_source: Info

figma2flutter-0.2.0-alpha

### :speech_balloon: Description

Getting an error message:
```
Found 8 themes, generating code
Unhandled exception:
Instance of 'ProcessTokenException'
#0      Processor._safeProcess (package:figma2flutter/processor.dart:49:7)
#1      Processor._loopProcess (package:figma2flutter/processor.dart:39:9)
#2      Processor.process (package:figma2flutter/processor.dart:23:22)
#3      _processTokens (file:///home/supper/.pub-cache/hosted/pub.dev/figma2flutter-0.2.0-alpha/bin/figma2flutter.dart:110:13)
#4      main (file:///home/supper/.pub-cache/hosted/pub.dev/figma2flutter-0.2.0-alpha/bin/figma2flutter.dart:72:20)
<asynchronous suspension>
```

### :scroll: tokens file you tried to import

https://filebin.net/2pe0vukfkflei2ts/designtokens.json

**Comment by gronka** [comment]
I was able to build with `dart compile exe bin/figma2flutter.dart`

I found the bug. I'll open a PR in a bit.

**Comment by freemansoft** [comment]
Closing as fixed with 0.3.0 improved error logging.  The json file link is no longer valid
